readme.md

```diff
@@ -40,9 +44,15 @@ Why does the following code snippet throw an error, and what would fix the error
 var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
+```
+Clark Answer: When I type this code into chrome's console, the error is "Cannot set property 'backgroundColor' of undefined". There is only one body in a webpage, so I simply deleted
+querySelectorAll("body") and changed it to the following:
 
-> Your answer...
+var body = document.body
+body.style.backgroundColor = "red";
 
+Now the chrome console states that the body is "red"!!!
+```
```

Comment:
:+1:

Yup, you can set the body color that way. But also important to know **why** the code in the question does not work...

`querySelectorAll` returns an array of DOM object, even if there is only one of them on the page (like `body`). So you can either fix the code as you did or do something like `document.querySelector( "body" )`.